src/main/java/com/ericsson/gerrit/plugins/eiffel/configuration/EiffelPluginConfiguration.java

```diff
@@ -46,7 +62,71 @@ public EiffelPluginConfiguration(final String pluginName, final NameKey project,
             throw new ExceptionInInitializerError(String.format("Can't read %s plugin configuration for project %s: %s",
                     pluginName, project.toString(), e.getMessage()));
         }
+        // Read plugin configuration
+        this.enabled = pluginConfig.getBoolean(ENABLED, false);
+        this.filter = pluginConfig.getString(FILTER);
```

Comment:
You could return directly here if plugin is not enabled, instead of inside the validation

Reply:
This cannot be done as the Constructor expects all the fields to have been initiated.

Reply:
That is put a 
```
if(!this.enabled){
return;
}
```

at line 67 instead of the if-statements on 76 and 88. It will be the same logic as the `validatePluginConfig` returns directly if the plugin is not enabled.